extensions/logging-json/runtime/pom.xml

```diff
@@ -37,6 +37,12 @@
             <plugin>
                 <groupId>io.quarkus</groupId>
                 <artifactId>quarkus-bootstrap-maven-plugin</artifactId>
+                <configuration>
+                    <runnerParentFirstArtifacts>
+                        <!-- Needed because org.jboss.logmanager.formatters.JsonFormatter loads javax.json.Json in it's constructor -->
+                        <runnerParentFirstArtifact>org.glassfish:jakarta.json</runnerParentFirstArtifact>
```

Comment:
When doing that, is it only effective for this artifact? Because if not, I'm worried we would break JSON-B support.

Reply:
There should be no breakage - although only this artifact will be loaded by the System ClassLoader, all other artifacts that depend on it **will** be able to access it as the `RunnerClassLoader` uses the System ClassLoader as it's parent